src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
```

Comment:
[CODE STYLE] The `Herbivore` constructor should not accept `health` as a parameter since the task specifies health is `100` by default. Only `name` and `hidden` should be parameters, with `hidden` defaulting to `false` as you have correctly implemented.